R/strat_ests_barplot.R

```diff
@@ -0,0 +1,81 @@
+#' Barplot method for `summary.seroincidence.by` objects
+#'
+#' @param object a `summary.seroincidence.by` object (generated by applying the `summary()` method to the output of [estimate_scr_by()]).
```

Comment:
```suggestion
#' @param object a `summary.seroincidence.by` object (generated by applying the 
`summary()` method to the output of [estimate_scr_by()]).
```
see `line_length_linter` annotation